### PR TITLE
[ODS-4610] Web Api Appsettings.json is missing default configurations

### PR DESCRIPTION
**Pull request by raghuramakrishna**

### CODE REVIEW

File: Application/EdFi.Ods.WebApi/appsettings.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
   },
   "AllowedHosts": "*",
   "BearerTokenTimeoutMinutes" : "30",
+  "SecurityMetadataCacheTimeoutMinutes": "0",
   "DefaultPageSizeLimit" : 500,
   "ApiSettings": {
     "Mode": "sandbox",
```
Comment (blmeyers) [inline Application/EdFi.Ods.WebApi/appsettings.json:19]: The OdsTokens key below doesn't seem to be used, and didn't exist in the previous release of the API.  Can it be removed as part of this?  Its also set up as an array here, but in the generated development it outputs as empty string.  Probably need to remove from both.

File: Application/EdFi.Ods.WebApi/appsettings.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
   },
   "AllowedHosts": "*",
   "BearerTokenTimeoutMinutes" : "30",
+  "SecurityMetadataCacheTimeoutMinutes": "0",
   "DefaultPageSizeLimit" : 500,
   "ApiSettings": {
```
Comment (blmeyers) [inline Application/EdFi.Ods.WebApi/appsettings.json:18]: Should MinimalTemplateSuffix and PopulatedTemplateSuffix defaults migrate up to here from the development file?  @vimayya Related, should we have packaged defaults for the Minimal/Populated TemplateScript?

**Review by blmeyers — CHANGES_REQUESTED** [review]
Working off https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/v5.0.1/Application/EdFi.Ods.WebApi/Web.Base.config for comparison, as that represents the defaults that were in place with the previous release.

File: Application/EdFi.Ods.WebApi/appsettings.json (hunk 2 above)
Comment (vimayya) [reply Application/EdFi.Ods.WebApi/appsettings.json:18]: @blmeyers , @raghuramakrishna  yes. I have noted that in the comments here ODS-4622, I would prefer to have default local connection strings as well.

File: Application/EdFi.Ods.WebApi/appsettings.json [hunk 3]
```diff
@@ -61,8 +62,7 @@
         "IsEnabled": false
       }
     ],
-    "ExcludedExtensions": [],
-    "BearerTokenTimeoutMinutes": ""
```
Comment (vimayya) [inline Application/EdFi.Ods.WebApi/appsettings.json:65]: Aren't we using BearerTokenTimeoutMinutes?

Comment (blmeyers) [reply Application/EdFi.Ods.WebApi/appsettings.json:65]: This one was a duplicate, the actual value is higher up.

File: Application/EdFi.Ods.WebApi/appsettings.json (hunk 1 above)
Comment (blmeyers) [reply Application/EdFi.Ods.WebApi/appsettings.json:19]: It looks like this is primarily used for deployment to drive which databases need to be generated.  However, this should be set to an empty string and not an array, the powershell is expecting it to be a semicolon separated string at the moment.

**Review by blmeyers — CHANGES_REQUESTED** [review]

File: Application/EdFi.Ods.WebApi/appsettings.json (hunk 1 above)
Comment (raghuramakrishna) [reply Application/EdFi.Ods.WebApi/appsettings.json:19]: updated to empty string

**Comment by blmeyers** [comment]
@raghuramakrishna Have you pushed up all your changes?  You've marked my comments as resolved or mentioned that you fixed the issue, but I'm not seeing the expected changes.

**Comment by raghuramakrishna** [comment]
Sorry I have pushed the change. You can review now.

File: Application/EdFi.Ods.WebApi/appsettings.json [hunk 4]
```diff
@@ -6,18 +6,23 @@
...
   },
   "ConnectionStrings": {
-    "EdFi_Ods": "",
-    "EdFi_Admin": "",
-    "EdFi_Security": "",
-    "EdFi_master": ""
+    "EdFi_Ods": "server=(local);trusted_connection=True;database=EdFi_{0};Application Name=EdFi.Ods.WebApi",
+    "EdFi_Admin": "server=(local);trusted_connection=True;database=EdFi_Admin;Application Name=EdFi.Ods.WebApi",
+    "EdFi_Security": "server=(local);trusted_connection=True;database=EdFi_Security;persist security info=True;Application Name=EdFi.Ods.WebApi",
+    "EdFi_Master": "server=(local);trusted_connection=True;database=master;Application Name=EdFi.Ods.WebApi"
```
[the remark this answers is not in the document]
Comment (blmeyers) [reply Application/EdFi.Ods.WebApi/appsettings.json:12]: You marked this as resolved which normally implies you made the change, but I'm not seeing any changes to this.

File: Application/EdFi.Ods.WebApi/appsettings.json [hunk 5]
```diff
@@ -6,18 +6,23 @@
...
+  "SecurityMetadataCacheTimeoutMinutes": "0",
   "DefaultPageSizeLimit" : 500,
   "ApiSettings": {
     "Mode": "sandbox",
     "Engine": "",
     "UseReverseProxyHeaders": false,
+	"MinimalTemplateSuffix": "Ods_Minimal_Template",
+	"PopulatedTemplateSuffix": "Ods_Populated_Template",
+	"MinimalTemplateScript": "EdFiMinimalTemplate",
+	"PopulatedTemplateScript": "GrandBend",
```
[the remark this answers is not in the document]
Comment (blmeyers) [reply Application/EdFi.Ods.WebApi/appsettings.json:25]: You marked this as resolved which normally implies you made the change, but I'm not seeing any changes to this.

**Review by blmeyers — CHANGES_REQUESTED** [review]

**Comment by raghuramakrishna** [comment]
@blmeyers Strange I did resolved it before, I have updated it again. Please review

**Review by blmeyers — APPROVED** [review]

**Pull request merged**